.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs
```

Comment:
```suggestion
logs/*
```

Reply:
agree, lets exclude all files in logs folder